public/static/locales/cy/landing.json

```diff
@@ -82,19 +82,16 @@
           "text": "Yswiriant Teithio i bobl dros 65 oed. Sut olwg sydd ar bolisi da",
           "href": "https://www.moneyadviceservice.org.uk/cy/articles/yswiriant-teithio-ar-gyfer-pobl-dros-65-oed-a-chyflyrau-meddygol"
         },
-        {
-          "text": "dewis y polisi a'r cwmpas cywir",
-          "href": "https://www.moneyadviceservice.org.uk/cy/articles/yswiriant-teithio-dewiswch-y-polisi-ar-yswiriant-iawn"
-        },
+
         {
           "text": "Coronafirws ac yswiriant teithio",
           "href": "https://www.moneyadviceservice.org.uk/cy/articles/coronafeirws-ac-yswiriant-teithio"
         }
       ]
     },
     "broker": {
-      "heading": "Angen help i siopa o gwmpas",
-      "content": "<section style='margin: 15px; font-size: 16px; line-height: 20px; color:#515151'><p>Os nad ydych wedi cael unrhyw ddyfynbrisiau eto ac ar ddechrau eich chwiliad am yswiriant teithio, mae gan Gymdeithas Brocer Yswiriant Prydain (BIBA) wasanaeth sy'n eich cysylltu â Brocer a all wneud y siopa o gwmpas i chi a dod o hyd i chi y fargen orau gan ddarparwyr yswiriant teithio prif ffrwd ac arbenigol</p><br/><p style='font-size: 13px;'><b>Cysylltwch:<b/><br/><br/></p><img src='/assets/Images/phone.svg' style='width: 20px;' /> <span style='font-size: 22px; color:  #18507a;'>&nbsp;<b>0123 456 7891<b/></span><p style='padding-left: 30px; margin-bottom: 30px;'> &#40;Mon - Fri 9:00 am to 5:30 pm&#41;</p><img src='/assets/Images/web.svg'/><span>&nbsp; <a  style=' color:#18507a; font-size: 16px;' href=''>Gwefan</a></span</section>"
```

Comment:
Do these styles need to be inline or can they be added to a stylesheet?

Reply:
There are no inline styles there anymore. Your copy is probably stale.

Reply:
I mean in the translation file. I realise we need to add markup in these files sometimes but I believe we can deal with the styles differently.

Reply:
Yes, here is my copy.
<img width="802" alt="welshStyles" src="https://user-images.githubusercontent.com/20474902/93447449-53216080-f8ca-11ea-82c6-bbf1623bf0b8.png">

Reply:
Oh yeah, my bad. I had an outdated copy. All good now! 👍